src/TestHelper/src/System/Diagnostics/RemoteExecutorTestBase.uap.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Windows.ApplicationModel;
+using Windows.Foundation.Collections;
+using Windows.ApplicationModel.AppService;
+
+namespace System.Diagnostics
+{
+    /// <summary>Base class used for all tests that need to spawn a remote process.</summary>
+    public abstract partial class RemoteExecutorTestBase : FileCleanupTestBase
+    {
+        protected static readonly string HostRunnerExecutableName = "xunit.runner.uap";
```

Comment:
I kept it because it is used in other test code like S.D.Process tests. as I mentioned in other comment I'll try to get rid of the fields not used in the test projects